src/components/Outcomes/Outcomes.js

```diff
@@ -46,7 +46,7 @@ const OutcomesRow = ({ projection, label, color }) => {
       <div>
         {projection.dateOverwhelmed
           ? formatUtcDate(projection.dateOverwhelmed)
-          : 'Not in the next 90 days'}
+          : 'Not in the next 30 days'}
```

Comment:
nit: since you exported the PROJECTIONS_TRUNCATION_DAYS constant, you could use that here.